src/trufont/windows/fontWindow.py

```diff
@@ -247,7 +247,8 @@ def _newTab(self, glyph=None):
     def save(self, path=None):
         if path is None:
             path = self._path
-        if path is None:
+        # XXX: Expand to handle more formats once converters can properly round-trip.
+        if path is None or not path.endswith(".tfont"):
```

Comment:
Can you just not set a path into the window if you're not gonna save to that path? Because that's really the point of _path. Besides, only native format should be a Ctrl+S target. Conversion to another format is potentially expensive/lossy so should not be done if not called for.

Reply:
So, do you want non-tfont import to be done specifically from an "import" menu entry with an "export" entry to go along with it?